.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Shouldn't this file be `.gitignore`d ?

Reply:
Yeah this should definitely be in `gitignore`